data/packrat/artifacts.json

```diff
@@ -0,0 +1,2052 @@
+            "regex": [
+              "(?i-mx:password.*)",
+              "(?i-mx:username.*)"
+            ]
+          },
+          {
+            "extraction_description": "searches for Email TO/FROM address",
+            "extraction_type": "Email addresses",
+            "regex": [
+              "(?i-mx:to:.*)",
+              "(?i-mx:from:.*)"
+            ]
+          }
+        ]
+      },
+      {
+        "filetypes": "chat_logs",
+        "path": "AppData",
+        "dir": "Tlen.pl",
+        "artifact": "*.jpg",
+        "description": "Tlen.pl sent images"
+      }
+    ]
+  },
+  {
+    "application": "trillian",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "logins",
+        "path": "AppData",
+        "dir": "Trillian",
+        "artifact": "accounts.ini",
+        "description": "Trillian saved usernames and passwords",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for Identification",
+            "extraction_type": "Identification",
+            "regex": [
+              "(?i-mx:.*=.*)"
+            ]
+          }
+        ]
+      },
+      {
+        "filetypes": "chat_logs",
+        "path": "AppData",
+        "dir": "Trillian",
+        "artifact": "*.log",
+        "description": "Trillian logs",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for credentials (USERNAMES/PASSWORDS)",
+            "extraction_type": "credentials",
+            "regex": [
+              "(?i-mx:password.*)",
+              "(?i-mx:username.*)"
+            ]
+          },
+          {
+            "extraction_description": "searches for Email TO/FROM address",
+            "extraction_type": "Email addresses",
+            "regex": [
+              "(?i-mx:to:.*)",
+              "(?i-mx:from:.*)"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "viber",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "database",
+        "path": "AppData",
+        "dir": "ViberPC",
+        "artifact": "viber.db",
+        "description": "All Contact's names, numbers, sms are saved from user's mobile",
+        "credential_type": "database"
+      },
+      {
+        "filetypes": "thumbs",
+        "path": "AppData",
+        "dir": "ViberPC",
+        "artifact": "Thumbs.db",
+        "description": "Viber's Contact's profile images in Thumbs.db file",
+        "credential_type": "image"
+      },
+      {
+        "filetypes": "images",
+        "path": "AppData",
+        "dir": "ViberPC",
+        "artifact": "*.jpg",
+        "description": "Collects all images of contacts and sent recieved",
+        "credential_type": "image"
+      }
+    ]
+  },
+  {
+    "application": "xchat",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "chat_logs",

```

Comment:
There's a linux module, so not an overlap, but should be noted: https://github.com/rapid7/metasploit-framework/blob/76954957c740525cff2db5a60bcf936b4ee06c42/modules/post/linux/gather/enum_xchat.rb